boards/st/nucleo_u385rg_q/doc/index.rst

```diff
@@ -186,6 +186,8 @@ Default Zephyr Peripheral Mapping:
 ----------------------------------
 
 - DAC1_OUT1 : PA4
+- FDCAN1_TX : PA12
+- FDCAN1_RX : PA11
```

Comment:
Does this board have a CAN transceiver? If not, it would be good to add this as a note in the documentation (see e.g. `boards/st/stm32f3_disco/doc/index.rst`).